tests/conftest.py

```diff
@@ -1232,11 +1234,10 @@ def mock_get_source_ip() -> Generator[_patch]:
 
 @pytest.fixture(autouse=True, scope="session")
 def translations_once() -> Generator[_patch]:
-    """Only load translations once per session.
+    """Only load translations once per module.
```

Comment:
this comment should still say "per session", right? this fixture's scope hasn't changed